Framework/DataHandling/src/SaveOpenGenieAscii.cpp

```diff
@@ -37,7 +37,7 @@ void SaveOpenGenieAscii::init() {
 
   // Declare required parameters, filename with ext {.his} and input
   // workspace
-  const std::vector<std::string> exts{".his", ".txt", ""};
+  const std::vector<std::string> exts{".his", ".txt"};
```

Comment:
Can you explain these changes? It it intentional to forbid the files without extension, that were allowed before?

Reply:
This was part of the working solution that I have accidentally committed! Thanks for picking this up I'll remove it now